apps/web/src/connectors/sso.ts

```diff
@@ -23,6 +23,19 @@ window.addEventListener("load", () => {
   }
 });
 
+function initiateBrowserSsoIfDocumentReady(code: string, state: string, lastpass: boolean) {
+  if (document.readyState === "complete") {
+    initiateBrowserSso(code, state, lastpass);
+  } else {
+    const interval = setInterval(() => {
+      if (document.readyState === "complete") {
+        clearInterval(interval);
+        initiateBrowserSso(code, state, lastpass);
+      }
+    }, 20);
+  }
+}
```

Comment:
```suggestion
function initiateBrowserSsoIfDocumentReady(code: string, state: string, lastpass: boolean) {
  if (document.readyState === "complete") {
    initiateBrowserSso(code, state, lastpass);
    return;
  } 

  const interval = setInterval(() => {
    if (document.readyState === "complete") {
      clearInterval(interval);
      initiateBrowserSso(code, state, lastpass);
    }
  }, 20);
}
```

Nit: You can avoid this `else` statement with an early return...

Less of a nit, but not blocking the PR because of this thought: Checking every `20ms` seems kind of unnecessarily quick.... I'd imagine you'd have a comparable result checking each `100ms` or some longer period of time...